src/block_cache/block_chunkcache.c

```diff
@@ -694,10 +698,23 @@ __wt_chunkcache_reconfig(WT_SESSION_IMPL *session, const char **cfg)
         __wt_spin_lock(session, &chunkcache->hashtable[i].bucket_lock);
         TAILQ_FOREACH_SAFE(chunk, WT_BUCKET_CHUNKS(chunkcache, i), next_chunk, chunk_tmp)
         {
-            if (__chunkcache_should_pin_chunk(session, chunk))
+            if (__chunkcache_should_pin_chunk(session, chunk)) {
+                /*
+                 * Increment the stat only when a chunk, which was initially unpinned, becomes
+                 * pinned.
+                 */
+                if (!F_ISSET(chunk, WT_CHUNK_PINNED))
+                    WT_STAT_CONN_INCR(session, chunk_cache_chunks_pinned);
                 F_SET(chunk, WT_CHUNK_PINNED);
-            else
+            } else {
+                /*
+                 * Decrement the stat only when a chunk, which was initially pinned, becomes
```

Comment:
Same as above here.

Reply:
Updated in [4e598ca](https://github.com/wiredtiger/wiredtiger/pull/9480/commits/4e598cae83e35364fe35a1a3437acc2428362386)